console-src/README.md

```diff
@@ -0,0 +1 @@
+# Kontist API Console
```

Comment:
anything worth putting here?

Reply:
good idea, I have added some description here: 64cc128557a419779882ed6414966a2261731940

Not sure if enough, I wanted to keep it quite simple for now